app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,80 @@
+<%= stylesheet_link_tag "recipe.css" %>
+  <div class="wrapper-container">
+    <h1>Recipe <%= @recipe.name %>
+    </h1>
+    <div class="recipe-container">
+      <div>
+        <h4>Preparation Time: <%= @recipe.preparation_time %>
+        </h4>
+        <h4>Cooking Time: <%= @recipe.cooking_time %>
+        </h4>
+        <h4>Steps go there...</h4>
+      </div>
+      <div class="table-cont">
+    
+      <%= form_for @recipe, url: public_toggle_user_recipe_path(@recipe), method: :patch, remote: true, authenticity_token: true, html: { class: 'toggle-form' } do |f| %>
+        <div class="slider-container">
+        <span class="switch-label">Public</span>
+        <label class="switch">
+          <%= f.check_box :public, { class: "toggle-switch" }, 0, 1 %>
```

Comment:
- [ ] When I change a recipe to either public or private and refresh the page or go to another page, it goes back to what it originally was. Please make sure that when I change the recipe status to public/private it stays that way